rasterio/crs.pyx

```diff
@@ -924,27 +924,19 @@ cdef class CRS:
         cdef OGRSpatialReferenceH osr_s = NULL
         cdef OGRSpatialReferenceH osr_o = NULL
         cdef CRS crs_o
-
+        cdef const char* options[2]
+        options[0] = b"IGNORE_DATA_AXIS_TO_SRS_AXIS_MAPPING=YES"
+        options[1] = NULL
         try:
             crs_o = CRS.from_user_input(other)
```

Comment:
I'm no expert in cython, but could we check if `other` is a CRS object before doing `from_user_input`?

Reply:
https://github.com/rasterio/rasterio/blob/850d64586e3072ecb80f4d942f36e584d084997a/rasterio/crs.pyx#L765-L766

Reply:
`from_user_input` checks if it is a CRS or if it could be a CRS.